homework/express-cat/modelsdb.js

```diff
@@ -0,0 +1,11 @@
+var mongoose = require('mongoose');
+
+
+var catSchema = mongoose.Schema({
+    name: String,
+    age: Number,
+    colors: [String]
+})
+var modd = mongoose.model('Cat', catSchema); // why do I need this line?
```

Comment:
You should export this model instead of the schema. Then in `catroutes.js` require the model directly:
```node
var Cat = require(path.join(__dirname, '../modelsdb);
```